packages/client/src/processes/layout/model/effects.ts

```diff
@@ -15,3 +18,23 @@ export const oauthSignInFx = createEffect<OauthSignInRequest, void, AxiosError>(
     await practicumApi.oauth.yandexCreate(oauthSignInRequest);
   }
 );
+
+export const setUserThemeFx = createEffect<
+  { theme: Theme; userId: number },
+  { theme: Theme; userId: number },
+  AxiosError
+>(async data => {
+  const res = await axios.post(ServerUrl + apiPath.setTheme, data);
+  return res.data;
+});
+
+export const updateThemeFx = createEffect<
+  UserResponse | null,
+  Theme,
+  AxiosError
+>(async user => {
+  if (user) {
+    const res = await axios.get(ServerUrl + apiPath.getTheme + '/' + user.id);
+    return res.data.theme;
+  }
```

Comment:
Фильтрацию необходимо делать через sample

```suggestion
    const res = await axios.get(ServerUrl + apiPath.getTheme + '/' + user.id);
    return res.data.theme;
```

Reply:
Спасибо! Сделал фильтрацию в sample.